client/wfnews-war/src/main/angular/src/app/components/contact-widget/contact-widget.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="contact-widget">
```

Comment:
We'll be adding a second widget for the UI for the watchlist. Might be good to ensure the container can support additional buttons